math/minuit2/inc/Minuit2/MnMachinePrecision.h

```diff
@@ -16,6 +16,13 @@ namespace ROOT {
 
    namespace Minuit2 {
 
+// compute machine precision only once
+struct MnStaticMachinePrecision {
+  MnStaticMachinePrecision();
+  double eps, eps2;
+};
+
+static const MnStaticMachinePrecision static_precision;
```

Comment:
If I'm not mistaken, this makes one instance for each translation unit that this file is included into. Is this header included frequently?

Reply:
I think it is included in several file of Minuit2.
I agree, I mentioned before, one should move the static declaration in a .cxx file and therefore also the constructor of the class MnMachinePrecision from the header to a .cxx. 
@HDembinski , can you do this ? Otherwise I can take your branch and update it